vendor/bat-native-ledger/src/bat/ledger/internal/database/database_contribution_queue.cc

```diff
@@ -164,7 +215,7 @@ void DatabaseContributionQueue::InsertOrUpdate(
 void DatabaseContributionQueue::OnInsertOrUpdate(
     ledger::DBCommandResponsePtr response,
     const std::string& queue_string,
-    const uint64_t id,
+    const std::string& id,
```

Comment:
I think that id is not needed anymore, as you assign `info->id` all the time now in Insert function, so you should have it in `queue_string`

Reply:
Good catch, fixed.